prost-build/src/code_generator.rs

```diff
@@ -411,15 +412,16 @@ impl<'a> CodeGenerator<'a> {
         self.append_doc();
         self.push_indent();
 
-        let btree_map = self
-            .config
-            .btree_map
-            .iter()
-            .any(|matcher| match_ident(matcher, msg_name, Some(field.name())));
-        let (annotation_ty, rust_ty) = if btree_map {
-            ("btree_map", "BTreeMap")
+        let btree_map = self.config.force_btree_map ||
```

Comment:
yeah this definitely shouldn't need to change.  At worst, the `force_btree_map` option can be implemented by setting `btree_map(["."])`, but I don't think it's pulling its weight as a standalone option.